Resources/Prototypes/Recipes/Reactions/pyrotechnic.yml

```diff
@@ -41,8 +41,9 @@
   - !type:ExplosionReactionEffect
     devastationRange: 0
     heavyImpactRange: 0
-    lightImpactRange: 2
-    scaled: false
+    lightImpactRange: 0.1
```

Comment:
FIY because the effect currently rounds the distance, this means there wont be any explosion until you have at least 5+15 units. But I guess that's fine?

Actually it still seems to deal damage even when the radius is zero, the lookup is just tiny. So it damage the person holding a beaker (or I guess anyone standing in a cloud of it). I think you probably just need to edit `ExplosionReactionEffect.cs` and add a lower-limit or something.

Reply:
I guess I'll do it the way you actually suggested